README.md

```diff
@@ -129,6 +131,14 @@ following keys:
 We welcome contributions to Conflation. If you would like to report an issue, or even better fix an existing one, please
 use the [Conflation issue tracker](https://github.com/OpenStreetMapSpeeds/conflation/issues) on GitHub.
 
+To install the project in development mode plus the needed libraries, do a `pip install -e ".[dev]"`.
+
+We encourage you to install the pre-commit hooks by typing `pre-commit install` which will run the following commands to lint and style-check your code before committing:
+```shell script
+flake8 .
+black .
+```
```

Comment:
A bit tangential, but @nilsnolde have you used this set of hooks with Pycharm? I can probably Google to see if there's a nice setup but wanted to check in case you've been through it before.

Reply:
Actually it seems I misunderstood. So we only need to run `pre-commit install` _once_ and then the hooks will run automatically on certain `git` actions?

Reply:
the `pre-commit install` will do that for you, independent of the IDE you're using

Reply:
it really just creates a `.git/hooks/pre-commit` script with the info from the yaml config

Reply:
> So we only need to run pre-commit install once and then the hooks will run automatically on certain git actions?

sorry, I was too fast :sweat_smile:  correct! only on `commit`. there's even hooks for `push` but I abandoned that for auto-testing before pushing, too annoying..

Reply:
Awesome! That'll be super convenient.